src/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -535,7 +535,7 @@ private static void CheckBaseAddress(Uri baseAddress, string parameterName)
 
         private void SetTimeout(CancellationTokenSource cancellationTokenSource)
         {
-            Contract.Requires(cancellationTokenSource != null);
+            Debug.Assert(cancellationTokenSource != null);
```

Comment:
Yeah, we can't for all of them, as there are other uses of Contract still, like Contract.Ensure, Contract.End*, etc.